app/controllers/users_controller.rb

```diff
@@ -1,5 +1,5 @@
 class UsersController < ApplicationController
   def personas
```

Comment:
We could have used `PersonasController#index` here instead of a funky action.

Reply:
I was thinking the same and even mentioned it to Alistair on Friday. Let's do it!

Reply:
That sounds lovely @jhenderson

Reply:
Yep. Agreed.